test.js

```diff
@@ -284,6 +284,24 @@ test('removes -nocookie', t => {
 	t.is(fn('http://www.youtube-nocookie.com/user/username#p/u/1/ABC12302').service, 'youtube');
 });
 
+/** Unfortunately I was not able to run the tests locally as there were
```

Comment:
I would rather not see a code comment about how you couldn't run the tests baked into the source file itself. We should probably work out the reasons why you couldn't run the tests and fix those. Where were all these syntax errors coming from?